bandit/reporters/crash_reporter.h

```diff
@@ -9,7 +9,7 @@ namespace bandit {
   namespace detail {
     struct crash_reporter : public progress_reporter {
       crash_reporter(std::ostream& stm, const failure_formatter& failure_formatter)
-          : progress_reporter(failure_formatter), stm_(stm), contexts_() {}
+          : progress_reporter(failure_formatter), stm_(stm), cr_contexts_() {}
```

Comment:
It seems that it would be a better change to get rid of this variable because it essentially mimics `contexts_` from `progress_reporter`. So the crash reporter can just use that one.